src/main/java/core/basesyntax/service/operationtypes/SupplyOperationHandler.java

```diff
@@ -1,11 +1,11 @@
-package core.basesyntax.service.operationTypes;
+package core.basesyntax.service.operationtypes;
 
 import core.basesyntax.dao.FruitsDaoImpl;
 import core.basesyntax.model.Fruit;
 
 //
-public class SupplyOperationHandler implements OperationTypeHandler{
-    FruitsDaoImpl fruitsDao = new FruitsDaoImpl();
+public class SupplyOperationHandler implements OperationTypeHandler {
+    private FruitsDaoImpl fruitsDao = new FruitsDaoImpl();
```

Comment:
FruitDaoImpl is realization and on the left we need abstraction FruitDao

Reply:
Agree, fixed